mne/preprocessing/infomax_.py

```diff
@@ -56,8 +56,13 @@ def infomax(data, weights=None, l_rate=None, block=None, w_change=1e-12,
         The window size for kurtosis estimation. Only considered for extended
         Infomax.
     ext_blocks : int
-        The number of blocks after which to recompute Kurtosis.
         Only considered for extended Infomax.
+        If positive, it denotes the number of blocks after which to recompute
+        the Kurtosis, which is used to estimate the signs of the sources.
+        In this case the number of sub-gaussians sources is automatically
+        determined.
+        If negative, the number of sub-gaussians sources to be used is fixed
```

Comment:
I think you mean "the number of sub-gaussian sources" not "the number of sub-gaussians sources"